samples/bluetooth/hci_spi/src/main.c

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2016 Nordic Semiconductor ASA
+ * Copyright (c) 2015-2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <stddef.h>
+#include <stdio.h>
+
+#include <zephyr.h>
+#include <misc/byteorder.h>
+#include <logging/sys_log.h>
+#include <misc/stack.h>
+
+#include <device.h>
+#include <init.h>
+#include <gpio.h>
+#include <spi.h>
+
+#include <net/buf.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/l2cap.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/buf.h>
+#include <bluetooth/hci_raw.h>
+
+#include "common/log.h"
+
+#define HCI_CMD			0x01
```

Comment:
nitpicking, general comment throughout in this file, use spaces to align macro values (instead of tabs as of now e.g. between HCI_CMD \t\t\t 0x01).